### PR TITLE
feat(DevTools): Pin properties

### LINKED ISSUE #12242: [DevTools] Pinned Property
**Is your feature request related to a problem? Please describe.**
When debugging with Devtools in some circumstances you need to keep the focus on only a few properties. As it is now if the properties have different names it is necessary to scroll the datagrid to see the value.

**Describe the solution you'd like**
Adding pinned properties like VS Debug Visualizer, which shows the pinned properties at the top of the property list.

**Describe alternatives you've considered**
Problem, where to store pinned properties for restore between debug session?

**Additional context**
none

### PR DESCRIPTION
**Pull request by workgroupengineering**
## What does the pull request do?

Adding pinned properties like VS Debug Visualizer, which shows the pinned properties at the top of the property list.

Pinned Properties

## Checklist

- [ ] Added unit tests (if possible)?
- [ ] Added XML documentation to any related classes?
- [ ] Consider submitting a PR to https://github.com/AvaloniaUI/avalonia-docs with user documentation

## Fixed issues

Fixes #12242
Closes  #12315

### CODE REVIEW

**Comment by maxkatz6** [comment]
I think it would be neat if pin icon appears on the pointer over.

File: src/Avalonia.Diagnostics/Assets/Icons.axaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<ResourceDictionary xmlns="https://github.com/avaloniaui"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
```
Comment (maxkatz6) [inline src/Avalonia.Diagnostics/Assets/Icons.axaml:2]: Add x:ClassModifier=internal

File: src/Avalonia.Diagnostics/Diagnostics/Views/ControlDetailsView.xaml [hunk 2]
```diff
@@ -1,19 +1,32 @@
 ﻿<UserControl xmlns="https://github.com/avaloniaui"
              xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-             xmlns:conv="using:Avalonia.Diagnostics.Converters"
-             xmlns:local="using:Avalonia.Diagnostics.Views"
-             xmlns:controls="using:Avalonia.Diagnostics.Controls"
-             xmlns:vm="using:Avalonia.Diagnostics.ViewModels"
-             xmlns:lb="using:Avalonia.Diagnostics.Behaviors"
+             xmlns:conv="clr-namespace:Avalonia.Diagnostics.Converters"
```
Comment (maxkatz6) [inline src/Avalonia.Diagnostics/Diagnostics/Views/ControlDetailsView.xaml:3]: Why clr-namespace?

Comment (workgroupengineering) [reply src/Avalonia.Diagnostics/Diagnostics/Views/ControlDetailsView.xaml:3]: Only for this issue #12974. I make PR https://github.com/kekekeks/XamlX/pull/94 , has not yet been approved.

Comment (maxkatz6) [reply src/Avalonia.Diagnostics/Diagnostics/Views/ControlDetailsView.xaml:3]: pinged about this PR

**Review by maxkatz6 — APPROVED** [review]

**Pull request merged**